app/routes/deck.js

```diff
@@ -1,6 +1,12 @@
 import Ember from 'ember';
 
 export default Ember.Route.extend({
+  beforeModel: function () {
```

Comment:
another future improvement will be to not show the "Create" or "Import" buttons unless they sign in